modules/login/templates/form_login.tpl

```diff
@@ -11,13 +11,13 @@
               <img src="{$baseurl}/{$study_logo}" alt="{$study_title}"/>
             </section>
           {/if}
-          <form method="POST" action="{$action}">
+          <form method="POST" action="">
```

Comment:
No reason to have the $action variable here and similar for the $username variable on line 16.